### ISSUE TITLE
Fix CircleCI Tests After Python 3.10.8 Update

### ISSUE THREAD

**Issue by okkays** [usdoj-crt/crt-portal-management#1451]
## Summary

We've [temporarily removed](https://github.com/usdoj-crt/crt-portal/commit/88e8d3c11f70cbe0484823578f1659d5aa07444d#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47) our automated pa11y and end-to-end (e2e) test checks on push. This means we're relying on testing locally, or before deployment, to detect e2e and accessibility failures.

### Steps To Reproduce

1. [Re-enable these tests](https://github.com/usdoj-crt/crt-portal/blob/develop/.circleci/config.yml#L120-L129) on a new branch.
2. Push that branch
3. CircleCI will fail at later steps.

### Expected Behavior

The tests should pass when enabled ([Example pass](https://app.circleci.com/pipelines/github/usdoj-crt/crt-portal/7449/workflows/48fc3171-5661-4184-88eb-ea2ff8a82df4/jobs/10089))

### Actual Behavior

The tests fail [(Example failure)](https://app.circleci.com/pipelines/github/usdoj-crt/crt-portal/7473/workflows/da373e8e-a6a4-4ae3-b1f9-822e0a31d72f/jobs/10131).

## Suggested Actions

Debug the CircleCI failures using SSH. A precursory glance indicates changes to Node's handling of ipv4 may be at fault, but more investigation is required.

## Acceptance Criteria

The tests pass while uncommented, ideally while using up-to-date versions of dependencies.
